packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1605,8 +1605,8 @@ abstract class RenderObjectElement<T extends RenderObjectWidget> extends Buildab
     while ((oldChildrenTop <= oldChildrenBottom) && (newChildrenTop <= newChildrenBottom)) {
```

Comment:
Yeah, the changes here shouldn't be harmful to them, but they don't all support detachChild yet.